src/mcp/mcp_utils.py

```
@@ -0,0 +1,62 @@
+    Returns the MCP server object, or raises an error if not found.
+    """
+    config = read_mcp_config()
+    mcp_servers = config.get("mcpServers", {})
+    mcp_server = mcp_servers.get(mcp_name)
+    try:
+        return ComposioMCP(
+            name=mcp_name,
+            url=mcp_server.get('url'),
+            description=mcp_server.get('description')
+        )
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)